packages/components/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -37,9 +37,9 @@ const StyledProgress = styled.div`
       theme.colors.colorBackgroundWeak
     )};
   border-radius: ${(props: StyledProgressProps) =>
-    get(theme.radii, String(props.radius), theme.radii.borderRadius10)};
+    get(theme.radii, String(props.radius), "")};
```

Comment:
Is this just passing an empty string? If so do we need it?

Reply:
This is so lodash get can infer that we're using strings here. Removing it would require me to add something like

```suggestion
    get(theme.radii, String(props.radius)) as string};
```

Reply:
The empty string here represents the default case but since radius is required this is what we get when the value is not found in theme.radii. Using the empty string also allows the style to fail instead failling silently by letting the developer pass a non existent radius here and the program still rendering everything perfectly